configs/common/modules_tcl.yaml

```diff
@@ -130,16 +130,14 @@ modules:
           ^esmf@8.5.0+debug snapshot=none: 'esmf-8.5.0-debug'
           ^esmf@8.6.0~debug snapshot=none: 'esmf-8.6.0'
           ^esmf@8.6.0+debug snapshot=none: 'esmf-8.6.0-debug'
-          ^esmf@8.6.1b04~debug snapshot=b04: 'esmf-8.6.1b04'
-          ^esmf@8.6.1b04+debug snapshot=b04: 'esmf-8.6.1b04-debug'
           ^esmf@8.6.1~debug snapshot=none: 'esmf-8.6.1'
           ^esmf@8.6.1+debug snapshot=none: 'esmf-8.6.1-debug'
-          ^esmf@8.7.0b04~debug snapshot=b04: 'esmf-8.7.0b04'
-          ^esmf@8.7.0b04+debug snapshot=b04: 'esmf-8.7.0b04-debug'
-          ^esmf@8.7.0b11~debug snapshot=b11: 'esmf-8.7.0b11'
-          ^esmf@8.7.0b11+debug snapshot=b11: 'esmf-8.7.0b11-debug'
           ^esmf@8.7.0~debug snapshot=none: 'esmf-8.7.0'
           ^esmf@8.7.0+debug snapshot=none: 'esmf-8.7.0-debug'
+          ^esmf@8.8.0~debug snapshot=none: 'esmf-8.8.0'
```

Comment:
Since beta tags lead up to the final release, it might be better (i.e. less confusing) to have the two `8.8.0` lines _after_ the `8.8.0b06` lines.

Reply:
Also I see that other parts in spack-stack will need updating once 8.8.0 is actually available... maybe don't list those lines here yet to prevent confusion about 8.8.0 availability?